frontend/talentsearch/src/js/components/GovernmentInfoForm/GovernmentInfoForm.tsx

```diff
@@ -495,9 +495,13 @@ export const GovInfoFormContainer: React.FunctionComponent = () => {
           const message = intl.formatMessage(profileMessages.profileCompleted);
           if (!preProfileStatus && currentProfileStatus) {
             toast.success(message);
+            navigate(paths.profile());
           }
           navigate(paths.profile());
           toast.success(intl.formatMessage(profileMessages.userUpdated));
+        } else {
+          navigate(paths.profile());
+          toast.success(intl.formatMessage(profileMessages.userUpdated));
```

Comment:
Is it possible to move the `toast` and `navigate` just after the first `if` statement to reduce the duplication occurring and no longer require an `else` statement?

Reply:
[like so?](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3175/commits/ac9605049024064029a72ffef3d686b634c0abc1)